nano/node/election.hpp

```diff
@@ -116,7 +116,10 @@ class election final : public std::enable_shared_from_this<nano::election>
 	void transition_active ();
 
 public: // Status
+	// Returns true when the winning block is durably confirmed in the ledger.
```

Comment:
From the comments in these 4 lines, I do not fully understand the difference between confirmed and status_confirmed. The comments need to be improved.